File: 1808 B. Playing in a Casino/Solution.py
#Krishno_MD_KUET_CSE
t = int(input())
for i in range(t):
    n,m = map(int,input().split())
    v = [list(map(int,input().split())) for i in range(n)]
    tot = 0
    for i in range(m):
        col = [0]*n
        for j in range(n):
            col[j] = v[j][i]
        col.sort()
        pre = 0
        for k in range(n):
            tot += k*col[k] - pre
            pre += col[k]
    print(tot)

File: 1808 B. Playing in a Casino/Solution.cpp
#include<bits/stdc++.h>
using namespace std; 
#define ll long long

// Krishno_MD_KUET_CSE

void solve(){
    int n,m;
    cin>>n>>m;
    vector<vector<int>> v(n,vector<int>(m));
    for(int i=0; i<n; i++){
        for(int j=0; j<m; j++){
            cin>>v[i][j];
        }
    }
    ll tot = 0;
    for(int i=0; i<m; i++){
        vector<int> col(n);
        for(int j=0; j<n; j++){
            col[j] = v[j][i];
        }
        sort(col.begin(),col.end());
        ll pre = 0;
        for(int k=0; k<n; k++){
            tot += k*(ll)col[k] - pre;
            pre += col[k];
        }
    }
    cout<<tot<<endl;
}

int main(){
    int t;
    cin >> t;
    while(t--){
        solve();
    }
    return 0;
}
